tests/integration/Renderer.test.ts

```diff
@@ -15,10 +15,25 @@ function isDir(dirPath: Dirent | string): boolean {
     }
 }
 
+function getSimpleImagesObj(x:Worksheet) {
+    return  x.getImages().map(({ imageId, range }) => (
+        {
+            imageId,
+            brc: (range.br as Anchor).nativeCol,
```

Comment:
casting will be redundant after https://github.com/exceljs/exceljs/pull/1356 was merged.

Reply:
exceljs/exceljs#1356 was merged, but I want to keep compatibility with older ExcelJS versions, so should use casting here